test/e2e-tests.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This script runs the end-to-end tests against the eventing sources
+# This script runs the end-to-end tests against the eventing
```

Comment:
'sources' was removed. Was that on purpose?

Reply:
I'm not sure anymore. I don't really know what this sentence means in knative/eventing. Probably not the same as what it means in knative/eventing-sources. I added `sources` back anyway.